apps/src/templates/rubrics/rubricShapes.jsx

```diff
@@ -19,6 +19,12 @@ export const rubricShape = PropTypes.shape({
     position: PropTypes.number,
     name: PropTypes.string,
   }),
+  script: PropTypes.shape({
+    id: PropTypes.number,
+  }),
+  level: PropTypes.shape({
+    id: PropTypes.number,
+  }),
```

Comment:
Now needed in order to support the teacherFeedback API for supporting setting progress state to 'keepWorking'